### PR TITLE
Add bridge configuration and parser for `Will message`

### PR DESCRIPTION
**Pull request by alvin1221**

### CODE REVIEW

**Review by wanghaEMQ — APPROVED** [review]

**Review by lee-emqx — APPROVED** [review]

**Review by JaylinYu — APPROVED** [review]
bridge.c?

File: demo/mqttv5/mqttv5_client.c [hunk 1]
```diff
@@ -207,7 +215,7 @@ static void
 send_callback(nng_mqtt_client *client, nng_msg *msg, void *arg) {
 	nng_aio *        aio    = client->send_aio;
 	uint32_t         count;
-	uint8_t *        code;
+	reason_code *    code;
```
Comment (wanghaEMQ) [inline demo/mqttv5/mqttv5_client.c:218]: Has the error of size of the reason code been fixed?

File: include/nng/mqtt/mqtt_client.h [hunk 2]
```diff
@@ -401,20 +401,22 @@ NNG_DECL void nng_mqtt_msg_set_connect_clean_session(nng_msg *, bool);
...
 NNG_DECL const char *nng_mqtt_msg_get_connect_client_id(nng_msg *);
 NNG_DECL const char *nng_mqtt_msg_get_connect_will_topic(nng_msg *);
-NNG_DECL uint8_t *   nng_mqtt_msg_get_connect_will_msg(nng_msg *, uint32_t *);
-NNG_DECL bool        nng_mqtt_msg_get_connect_will_retain(nng_msg *);
-NNG_DECL uint8_t     nng_mqtt_msg_get_connect_will_qos(nng_msg *);
-NNG_DECL property   *nng_mqtt_msg_get_connect_property(nng_msg *);
+NNG_DECL uint8_t *nng_mqtt_msg_get_connect_will_msg(nng_msg *, uint32_t *);
+NNG_DECL bool     nng_mqtt_msg_get_connect_will_retain(nng_msg *);
+NNG_DECL uint8_t  nng_mqtt_msg_get_connect_will_qos(nng_msg *);
+NNG_DECL property *nng_mqtt_msg_get_connect_property(nng_msg *);
```
Comment (wanghaEMQ) [inline include/nng/mqtt/mqtt_client.h:419]: Is your format tool broken?

File: demo/mqttv5/mqttv5_client.c (hunk 1 above)
Comment (JaylinYu) [reply demo/mqttv5/mqttv5_client.c:218]: that PR is not yet merged

**Pull request merged**